src/App.css

```diff
@@ -5,6 +5,5 @@
 .Content {
   display: flex;
   flex-direction: column;
-  text-align: center;
-  justify-content: center;
+  align-items: center;
 }
```

Comment:
maybe you are going to do it anyway but perhaps create a margin-top? :)

Reply:
Yes you are right. I only put in the bare minimum for styling and will put some attention to that stuff later on:)